static/app/views/performance/transactionSummary/transactionOverview/content.tsx

```diff
@@ -344,6 +378,83 @@ function SummaryContent({
     organization.features.includes('performance-spans-new-ui') &&
     organization.features.includes('insights-initial-modules');
 
+  // This is adapted from the `getEventFieldValues` function in `events/searchBar.tsx`
+  const getTransactionFilterTagValues = memoize(
```

Comment:
There's no need to use `memoize` anymore, the component should cache these results automatically (since it uses `useQuery` under the hood)

Reply:
Oh right! Good callout, I'll change that